app/src/Components/DisplayCards/DisplayCards.css

```diff
@@ -46,13 +48,14 @@
   text-decoration: none;
   background-color: #ED1B24;
   margin: .2rem;
-  padding: 0.5rem 0.75rem;
+  padding: 0.5rem 0.75rem 0.6rem 0.75rem;
   border-radius: 0.4rem;
 }
 
 .pagination-btn a:hover {
   color: white;
   font-weight: 800;
+  font-size: large;
```

Comment:
I noticed that when you hover over the pagination buttons, the whole button size changes. This then shifts the entire page down a few pixels. Is there a way to fix this? If you can't, you can always just leave it the same size instead of making the button text bigger.